module/data/vars.py

```diff
@@ -1,3 +1,23 @@
+import enum
+from enum import Enum
+
+
+# Translation IDs
+class TEXT_IDS(Enum):
+    START_TEXT_ID: int = enum.auto()
+
+
+# Translation Dictionary
+translations: dict[str, dict[int, str]] = {
+    "it": {
+        TEXT_IDS.START_TEXT_ID: "Benvenuto! Questo bot è stato realizzato dagli studenti del Corso di Laurea in Informatica al fine di suppotare gli studenti del DMI!\nPer scoprire cosa puoi fare usa /help o premi il pulsante ❔ Help"
+    },
+    "en": {
+        TEXT_IDS.START_TEXT_ID: "Welcome! This bot is developed by Computer Science's students to help fellow DMI's students!\nUse /help or press the ❔ Help button to know what this bot can do"
+    }
+}
```

Comment:
I don't really like this approach :S
We could create a yaml or json file like `it.json`, `en.json` .
So, if tomorrow wants to make `es.json` it's easy to copy-paste the en.json and translate it.

Reply:
Moreover, it's not important, but we are loosing the possibility to change the .md files while the bot is running, in this way we need to restart the bot to reload the texts

Reply:
> Moreover, it's not important, but we are loosing the possibility to change the .md files while the bot is running, in this way we need to restart the bot to reload the texts

I didn't know about this issue. Well, should be base the translation on md files and convert the get_locale into a path resolver? xD

Reply:
Hmm maybe we can use json files and hot-read them everytime we need to emit a text?